### PR TITLE
master-lps-136046

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-136046

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/json/trash/JSONRecycleBin.macro [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+		if (!(isSet(userEmailAddress))) {
+			var userEmailAddress = "test@liferay.com";
+		}
+
+		if (!(isSet(userPassword))) {
+			var userPassword = "test";
+		}
+
+		var groupId = JSONGroupAPI._getGroupIdByName(groupName = "${groupName}", site = "${site}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/trash/JSONRecycleBin.macro:23]: var groupId = JSONGroupAPI._getGroupIdByName(groupName = "${groupName}", site = "${site}", parentGroupName = "${parentGroupName}", grandParentGroupName = "${grandParentGroupName}");

File: portal-web/test/functional/com/liferay/portalweb/macros/json/trash/JSONRecycleBin.macro [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+		var curl = '''
+			${portalURL}/api/jsonws/trash.trashentry/delete-entries \
+			-u ${userEmailAddress}:${userPassword} \
+			-d groupId=${groupId}
+		''';
+
+		var messageId = JSONCurlUtil.post("${curl}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/trash/JSONRecycleBin.macro:31]: JSONCurlUtil.post("${curl}");

**Review by Tim-Cao — COMMENTED** [review]
Hi @Jeremy-Ch, Looks good enough

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**